src/main/java/minicraft/entity/Axis2D.java

```diff
@@ -0,0 +1,12 @@
+package minicraft.entity;
+
+public enum Axis2D {
+	X, Y;
+
+	public Direction getDirection(int displacement) {
+		if (this == X)
+			return Direction.getDirection(displacement, 0);
+		else
+			return Direction.getDirection(0, displacement);
+	}
+}
```

Comment:
This is a case where it would be better to use a function instead of a class. Let's try to avoid the excessive use of OOP.